neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jDatabaseMetaData.java

```diff
@@ -788,48 +792,50 @@ private String getIndexesV3X(boolean unique, boolean hasTable) {
 		queryBuilder.append("UNWIND range(0, size(properties) - 1) AS index\n");
 		queryBuilder.append("RETURN null AS TABLE_CAT,\n");
 		queryBuilder.append("null AS TABLE_SCHEM,\n");
-		queryBuilder.append("tokenNames[0] AS TABLE_NAME,\n");
+		queryBuilder.append("tokenNames AS TABLE_NAME,\n");
 		queryBuilder.append("type <> 'node_unique_property' AS NON_UNIQUE,\n");
 		queryBuilder.append("description AS INDEX_QUALIFIER,\n");
 		queryBuilder.append("description AS INDEX_NAME,\n");
 		queryBuilder.append(DatabaseMetaData.tableIndexOther + " AS TYPE,\n");
-		queryBuilder.append("index + 1 AS ORDINAL_POSITION,\n");
-		queryBuilder.append("properties[index] AS COLUMN_NAME,\n");
+		queryBuilder.append("0 AS ORDINAL_POSITION,\n");
+		queryBuilder.append("properties AS COLUMN_NAME,\n");
 		queryBuilder.append("null AS ASC_OR_DESC,\n");
 		queryBuilder.append("null AS CARDINALITY,\n");
 		queryBuilder.append("null AS PAGES,\n");
 		queryBuilder.append("null AS FILTER_CONDITION");
 		return queryBuilder.toString();
 	}
 
-	private String getIndexesV4X(String table, boolean unique, boolean hasTable) {
+	private String getIndexesV4X(boolean unique, boolean hasTable) {
```

Comment:
nit: maybe rename it just `getIndexes` since there is some v5 in it now

Reply:
I left it as is because is syntax mostly introduced in v4 but I'll change it